src/utils/utils.ts

```diff
@@ -88,9 +88,13 @@ export function isError(obj: any): obj is Error {
   return obj instanceof Error || (obj && obj.__proto__ && obj.__proto__.name === 'Error');
 }
 
-const isInDebugMode = !!getFromENV('PWDEBUG');
-export function isDebugMode(): boolean {
-  return isInDebugMode;
+const debugEnv = getFromENV('PWDEBUG') || '';
+export function debugMode() {
+  if (debugEnv === 'console')
```

Comment:
I personally would have created an enum for it. Maybe too complexity for this use-case.

Reply:
The result of this function is actually an enum 😄